.tekton/go-lint-pipelinerun.yaml

```diff
@@ -47,6 +47,8 @@ spec:
             value: .
           - name: version
             value: "v1.55"
+          - name: flags
+            value: --timeout=5m
```

Comment:
Why not to set in directly in `.golangci.yaml`?

Reply:
Done